build.rs

```diff
@@ -642,10 +667,11 @@ fn cc(
     c
 }
 
-fn nasm(file: &Path, arch: &str, out_file: &Path, include_dir: &Path) -> Command {
+fn win_asm(file: &Path, arch: &str, out_file: &Path, include_dir: &Path) -> Command {
```

Comment:
If you take my suggestion above about creating the `asm` function, then you can undo the changes to this function.